mldsa/poly.c

```diff
@@ -447,6 +447,11 @@ void poly_uniform_eta(poly *a, const uint8_t seed[MLDSA_CRHBYTES],
   ctr = rej_eta(a->coeffs, MLDSA_N, 0, buf, buflen);
   buflen = STREAM256_BLOCKBYTES;
   while (ctr < MLDSA_N)
+  __loop__(
+    assigns(ctr, state, memory_slice(a, sizeof(poly)), object_whole(buf))
+    invariant(ctr <= MLDSA_N)
+    invariant((&state)->pos <= SHAKE256_RATE)
```

Comment:
Nit: `state.pos` instead of `(&state)->pos`

Reply:
hmm, I get `/mldsa-native/mldsa/poly.c:450:1: error: member operator requires structure type on left hand side but got 'stream256_state *'` when I try this.

Reply:
Fixed [fdd5ca9](https://github.com/pq-code-package/mldsa-native/pull/224/commits/fdd5ca90f244729b2c831aefe029e15f129ccbb3)